client/src/pages/component/ContactUs/ContactUs.js

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import "./ContactUs.css";
+import { FaGithub, FaLinkedin } from "react-icons/fa";
+function ContactUs() {
+	return (
+		<>
```

Comment:
There doesn't need to be a `<>` and `</>` surrounding blocks like this where there is already a single root element. (This doesn't need to be changed, just a note!)